ivy_tests/test_ivy/test_frontends/test_paddle/test_tensor/test_paddle_linalg.py

```diff
@@ -606,3 +606,55 @@ def test_paddle_matrix_power(
         x=x[0],
         n=n,
     )
+
+
+# cond
+@st.composite
+def _get_dtype_and_matrix_non_singular(draw):
+    [None, "fro", "nuc", np.inf, -np.inf, 1, -1, 2, -2]
+    while True:
+        matrix = draw(
+            helpers.dtype_and_values(
+                available_dtypes=(
+                    ivy.float64,
+                    ivy.double,
```

Comment:
`ivy.double` is an alias of `float63` I think we will only need to specified exact supported dtypes for the function.
Refer to the Docs here https://www.paddlepaddle.org.cn/documentation/docs/en/2.5rc1/api/paddle/linalg/cond_en.html#cond

Reply:
Also, it would be better to not explicitly call ivy when testing frontend functions🙂

Reply:
> Also, it would be better to not explicitly call ivy when testing frontend functio

This makes sense